python/lsst/pipe/base/argumentParser.py

```diff
@@ -509,7 +509,12 @@ def __init__(self, name, usage="%(prog)s input [options]", **kwargs):
             help="logging level; supported levels are [trace|debug|info|warn|error|fatal]",
             metavar="LEVEL|COMPONENT=LEVEL",
         )
-        self.add_argument("--longlog", action=LongLogAction, help="use a more verbose format for the logging")
+        self.add_argument(
+            "--longlog",
+            nargs=0,
```

Comment:
I'm guessing this broke in DM-19634 (#178).

Reply:
We have no expectations that this will survive past the next weekly.